charts/grafana/templates/statefulset.yaml

```
@@ -46,6 +46,7 @@ spec:
     spec:
       accessModes: {{ .Values.persistence.accessModes }}
       storageClassName: {{ .Values.persistence.storageClassName }}
+      volumeName: {{ .Values.persistence.volumeName | quote }}
```

I guess it should be optional.

```suggestion
      {{- with .Values.persistence.volumeName }}
      volumeName: {{ . | quote }}
      {{- end }}
```

And please add the value in the values.yaml with docu as well.

---

Thank you for the review. 
Changes applied as suggested. Let me know if further adjustments are needed.